### PR TITLE
Add information about Chrome 108 permissions changes.

### PR DESCRIPTION
**Pull request by jpmedley**
This puts an [item from Known Issues](https://developer.chrome.com/docs/extensions/mv3/known-issues/#webrequest-onauthrequired) in the API reference.

### CODE REVIEW

File: site/en/docs/extensions/reference/webRequest/index.md
Comment (oliverdunk) [inline site/en/docs/extensions/reference/webRequest/index.md:24]: Would it make sense to move the sentence about that to a separate paragraph? While everything here is technically correct I feel like it could be read as implying that only policy-installed extensions can use `webRequestAuthProvider`, when in reality it's available to all extensions.

**Review by oliverdunk — APPROVED** [review]
Happy from a technical review POV. @jpmedley, I'm not sure if it makes sense to re-assign this back to you so I'll let you decide where it goes next :)

**Comment by jpmedley** [comment]
>Happy from a technical review POV. @jpmedley, I'm not sure if it makes sense to re-assign this back to you so I'll let you decide where it goes next :)

I din't go into enough detail about process yesterday. I apologize. 
1. Use GitHub's approval feature when you're done. One of the reasons our throughput is bad is that we're not giving clear signals that something is done. 
1. After your approval, assign it back to me for the typo review.
1. After I approve it, the PR creator merges it. (this is standard practice in Chrome DevRel. Sometimes articles have to publish on a particular day. The PR creator keeps track of that better than anyone else.

Also, we should all know the immutable law of text reviews: **There's always something to fix.**

No matter how many times you go through a piece of writing, there will always be something to fix. 

Therefore:

Unless you find a problem that's particularly severe (it brings down the apocalypse. Dogs and cats living together... MASS HYSTERIA!) open a ticket and move on with the review.

File: site/en/docs/extensions/reference/webRequest/index.md
Comment (AmySteam) [inline site/en/docs/extensions/reference/webRequest/index.md:337]:
```suggestion
As this function uses a blocking event handler, it requires the `"webRequest"` as well as the
```

**Review by AmySteam — APPROVED** [review]
LGTM, just a quick suggestion and should be good to go 👍

**Pull request merged**
